src/identifiers/forms.py

```diff
@@ -42,8 +42,6 @@ def clean(self):
         idents = models.Identifier.objects.filter(
             id_type=id_type,
             identifier=identifier,
-        ).exclude(
-            article=self.article,
```

Comment:
This will make it so that identifier edits won't be valid (since the query will return the identifier being edited)
A solution to this would be:

```
if identifier:
    idents = idents.exclude(id=identifier.id)
```